tox.ini

```diff
@@ -1,19 +1,28 @@
 [tox]
 envlist =
     py{27,35}-codestyle,
-    py{27,35}-django{18,19,110}
+    py{27,35}-django{18,19,110,111}
 
 [testenv]
 setenv =
     PYTHONDONTWRITEBYTECODE=1
-install_command = pip install --no-deps {opts} {packages}
+install_command = pip install {opts} {packages}
```

Comment:
The `--no-deps` parameter is to ensure all the deps are frozen in `requirements.txt` for repeatable builds, just put `pytz` in `requirements.in` and re-freeze with `pip-compile` 👍